src/server.js

```diff
@@ -291,11 +291,12 @@ function setupRest(app) {
 								server.uri.indexOf('turn:') !== 0 &&
 								server.uri.indexOf('turns:') !== 0))
 							continue;
-						let link = '<' + server.uri + '>; rel="ice-server";';
+						let link = '<' + server.uri + '>; rel="ice-server"';
 						if(server.username && server.credential) {
+							link += ';'
 							link += ' username="' + server.username + '";' +
 								' credential="' + server.credential + '";' +
-								' credential-type: "password";';
+								' credential-type= "password"';
```

Comment:
Ah good catch, ugly typo. Should the space after the equal be removed too, for consistency, since it's not in the others?